TheMET/TheMET/Network/MetAPI.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  MetAPI.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 25/04/2023.
+//
+
+import Foundation
+
+class MetAPI{
+    
+    private var networkManager: NetworkManager
+    
+    private let urlBaseString: String = "https://collectionapi.metmuseum.org/"
+    
+    init(networkManager: NetworkManager) {
+        self.networkManager = networkManager
+    }
+    
+    func objects(metadataDate: Date? = nil, departmentIds: [Int] = [], completion: @escaping (ObjectsResponse?) -> Void) {
+        var urlString: String = self.urlBaseString
+        let urlStringSuffix: String = "public/collection/v1/objects"
+        urlString.append(urlStringSuffix)
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd"
+        var parameters: [String : String] = [ : ]
+        let dateString: String?
+        if let date = metadataDate {
+            dateString = dateFormatter.string(from: date)
+        } else {
+            dateString = nil
+        }
+        let idArrayString: String?
+        if !departmentIds.isEmpty {
+            idArrayString = departmentIds.map {String($0)}.joined(separator: "|")
```

Comment:
instead of using an `idarraystring: String?`, just create an non-optional `idArrayString` at this line